scripts/azure-pipelines/linux-build_and_test.sh

```diff
@@ -16,21 +18,40 @@ function or_die () {
     fi
 }
 
-or_die mkdir azure-${TEST_TARGET}-build
-cd azure-${TEST_TARGET}-build
-if [[ "$DO_BUILD" == "yes" ]] ; then
-    or_die cmake -DCMAKE_CXX_COMPILER="${COMPILER}" ${CMAKE_EXTRA_FLAGS} ../src
-    if [[ ${CMAKE_EXTRA_FLAGS} == *COVERAGE* ]] ; then
-      or_die make -j 3
-    else
-      or_die make -j 3 VERBOSE=1
-    fi
-    if [[ "${DO_TEST}" == "yes" ]] ; then
-      or_die ctest -T test --output-on-failure -V
-    fi
-    if [[ "${DO_MEMCHECK}" == "yes" ]] ; then
-      or_die ctest -T memcheck
-    fi
-fi
+or_die cd axom
+git submodule init 
+git submodule update 
+
+echo HOST_CONFIG
+echo $HOST_CONFIG
+
+./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -DENABLE_GTEST_DEATH_TESTS=ON
+
+or_die cd build-$HOST_CONFIG-debug
+or_die make -j 3
+make CTEST_OUTPUT_ON_FAILURE=1 test ARGS='-T Test -VV -j8'
+
+
+find ./axom/sidre -type d -exec chmod 755 {} \;
+find ./axom/sidre -type f -exec chmod 644 {} \;
+find . -name "*.xml"
+
+#set -x
```

Comment:
@keithhealy -- should we fold the ``DO_BUILD``, ``DO_TEST`` logic back into the above script and delete the stuff below?

Reply:
Sorry I didn't clean up this file very well.
The find/chmod commands were necessary because some of the sidre files didn't have their permissions set correctly.  The 'find *.xml' is for my own debugging and I'll remove it along with the commented out code.

Reply:
As for the DO_BUILD and DO_TEST, I assumed there wouldn't be a case where we didn't want to do both (build and test) so I removed the logic.  I can restore it if that is preferable.

Again I apologize for not doing a better job cleaning up this file.

Reply:
Not a problem.  I agree that this seems unnecessary.

@white238, or others, do we need ``DO_BUILD`` and ``DO_TEST`` for our linux CI builds?
Perhaps we had it there to support our coverage tests? (which we'd like to add back in the near future).

Reply:
I think we need those since we will be able to do build tests for some compilers but may not be able to run tests; e.g., CUDA.

Reply:
Restored the DO_BUILD and DO_TEST logic.